graphql/schema.graphql

```diff
@@ -67,7 +67,10 @@ type Query {
 }
 
 type Mutation {
-  sessionWatcher(active: Boolean!): Boolean @cacheControl(scope: PRIVATE)
+  sessionWatcher(
+    active: Boolean!
+    regionalizationType: regionalizationTypeInput!
```

Comment:
Making it required will be a breaking change. Please consider an optional parameter with internal conditional so it will remain compatible with current implementations

Reply:
done, I have also included the logic for undefined variables on the sessionWatcher object for existing customers. 
It is node/resolvers/routes/index.ts lines 445 and 446